deployment/v3/external/kafka/README.md

```diff
@@ -8,6 +8,81 @@
 $ ./install.sh
 ```
 
+## Procedure to Backup & Restore Bitnami Kafka via Manually
+
+#### Prerequisites
+* Zip & Unzip packages to installed on both backup/restore
+
+#### Backup
+* Set websub and Kafka replicas to zero via the below command.
+  ```
+  kc2 -n default scale --replicas=0 deploy consolidator-websub-service websub-service
+  kc2 -n default scale --replicas=0 statefulset kafka kafka-zookeeper
+  ```
+* Once websub & kafka pod are completely terminated, Goto `/srv/nfs/mosip/`.
+  ```
+  cd /srv/nfs/mosip
+  ```
+* Run the below command to zip DMZ Kafka directories.
+  ```
+  sudo zip -r dmzkafka-backup.zip $( ls -d $( kc2 get pv | awk '/kafka/{print "*" $1}'))
+  ```
+* Move the backup file to the machine where the restore cluster.
+
+#### Restore
+
+* Deploy kafka on the restore cluster.
+* Run the below command to remove k8s probes and set pod command to `bash -c sleep infinity`.
+  ```
+  kubectl -n kafka patch statefulset kafka-zookeeper kafka --type=json \
+  -p='[
+        {"op": "replace", "path": "/spec/template/spec/containers/0/command", "value": ["bash"]},
+        {"op": "replace", "path": "/spec/template/spec/containers/0/args", "value": ["-c", "sleep infinity"]},
+        {"op": "replace", "path": "/spec/template/spec/containers/0/livenessProbe", "value": null},
+        {"op": "replace", "path": "/spec/template/spec/containers/0/readinessProbe", "value": null},
+        {"op": "replace", "path": "/spec/template/spec/containers/0/startupProbe", "value": null}
+  ]'
+  ```
+* Wait till all changes applied & pods are up and running.
+* Update N value (i.e., number of replicas ) in the below command, to clear the data.
+  ```
+  for i in {0..N}; do kubectl -n kafka exec -it kafka-zookeeper-$i -- bash -c 'rm -rf /bitnami/zookeeper/data/*'; done
+  for i in {0..N}; do kubectl -n kafka exec -it kafka-$i -- bash -c 'rm -rf /bitnami/kafka/data/*'; done
+  ```
+* Unzip the backup zip file via below command
+  ```
+  unzip dmzkafka-backup.zip  -d kafka/
+  
+  chown -R 1001:1001 kafka
+  ```
+* Update N value (i.e., number of replicas ) in the below command, to copy data from backup.
```

Comment:
To copy data from backup, update N value (i.e., number of replicas) in the below command.